Production/test/condorSub/dict_Summer20UL16APV_qcd.py

```diff
@@ -1,6 +1,7 @@
 flist = {
 
    "scenario": "Summer20UL16APV",
+   "args": "boostedsemivisible=True,emerging=True",
```

Comment:
These should be space separated lists of arguments. So for instance, this line should be:
```python
"args": "boostedsemivisible=True emerging=True",
```
The same comment holds for all of the "args" lines with more than one argument.

Reply:
Ah, I got it now, updated